SQL/0000-00-00-schema.sql

```diff
@@ -1643,6 +1671,386 @@ CREATE TABLE `parameter_type_override` (
+/*Global*/                                                                      
+CREATE TABLE `biobank_reference_table` (                                        
+  `ReferenceTableID` integer unsigned NOT NULL AUTO_INCREMENT,                  
+  `TableName` varchar(50) NOT NULL,                                             
+  `ColumnName` varchar(50) NOT NULL,                                            
+  CONSTRAINT `PK_biobank_reference_table` PRIMARY KEY (`ReferenceTableID`),     
+  CONSTRAINT `UK_biobank_reference_table_TableName_ColumnName` UNIQUE(`TableName`, `ColumnName`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;                                        
+                                                                                
+CREATE TABLE `biobank_datatype` (                                               
+  `DatatypeID` integer unsigned NOT NULL AUTO_INCREMENT,                        
+  `Datatype` varchar(20) NOT NULL UNIQUE,                                       
+  CONSTRAINT `PK_biobank_datatype` PRIMARY KEY (`DatatypeID`)                   
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;                                        
+                                                                                
+CREATE TABLE `biobank_unit` (                                                   
+  `UnitID` integer unsigned NOT NULL AUTO_INCREMENT,                            
+  `Label` varchar(20) NOT NULL UNIQUE,                                          
+  CONSTRAINT `PK_biobank_unit` PRIMARY KEY (`UnitID`)                           
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;                                        
+                                                                                
+                                                                                
+/*Container*/                                                                   
+                                                                                
+CREATE TABLE `biobank_container_capacity` (                                     
+  `ContainerCapacityID` integer unsigned NOT NULL AUTO_INCREMENT,               
+  `Quantity` decimal(10, 3) NOT NULL,                                           
+  `UnitID` integer unsigned NOT NULL,                                           
+  CONSTRAINT `PK_biobank_container_capacity` PRIMARY KEY (`ContainerCapacityID`),
+  CONSTRAINT `FK_biobank_container_capacity_UnitID`                             
+    FOREIGN KEY (`UnitID`) REFERENCES `biobank_unit`(`UnitID`)                  
+    ON UPDATE RESTRICT ON DELETE RESTRICT,                                      
+  CONSTRAINT `UK_biobank_container_capacity_Quantity_UnitID` UNIQUE(`Quantity`, `UnitID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;                                        
+                                                                                
+CREATE TABLE `biobank_container_dimension` (                                    
```

Comment:
I think this should be in the biobank_container_type directly. Does container types need to share common dimensions ?

Reply:
I think in the case of test-tubes there will be many containers that share the same dimension of 1:1:1

Reply:
It is in a different table to reduce row sizes?

Reply:
I think the design choice it was a question of reducing redundancy.

Reply:
same as capacity, I think that was part of avoiding repetition. I would not mind keeping this one separate just for aesthetics, I feel like these fields would just uglify the container_type table.... but im not super attached to it